getting-started/kubernetes/helm.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Installing Calico with Helm on Kubernetes
+description: Install Calico on a Kubernetes cluster using Helm 3.
+canonical_url: '/getting-started/kubernetes/helm'
+---
+
+### Big picture
+
+This installs Calico on a Kubernetes cluster using Helm 3.
+
+### Value
+
+Helm charts are a way to package up an application for Kubernetes.  This can be thought of as similar to the way `apt` or `yum` packages applications for operating systems.  
+
+Helm is also used by tools like ArgoCD to manage applications in a cluster, taking care of install, upgrade (and rollback if needed), etc. 
+
+### Before you begin
+
+- Install Helm 3.
+
+- Make sure you have a kubernetes cluster that meets the following requirements:
+  - kubernetes installed, but currently *without* a CNI plugin **OR** a cluster running a compatible CNI for Calico to run in policy-only mode 
+  - x86-64, arm64, ppc64le, or s390x processors
+  - RedHat Enterprise Linux 7.x+, CentOS 7.x+, Ubuntu 16.04+, or Debian 9.x+
+
+- Configure `kubeconfig` to work with your cluster (check by running `kubectl get nodes`)
+
+- Ensure that {{site.prodname}} can manage `cali` and `tunl` interfaces on the hosts.
+  If NetworkManager is present on the hosts, refer to
+  [Configure NetworkManager](../../maintenance/troubleshoot/troubleshooting#configure-networkmanager).
+
+### Concepts
+
+#### Operator based installation
+
+This guide uses Helm 3 to install the Tigera operator which will install {{site.prodname}}. The operator provides lifecycle management for Calico
```

Comment:
This guide uses Helm 3 to install the Tigera operator to install {{site.prodname}}. The operator provides lifecycle management for {{site.prodname}}, and is
exposed via the Kubernetes API and defined as a custom resource definition.

Reply:
```suggestion
In this guide, you install the Tigera {{site.prodname}} operator and custom resource definitions using the Helm 3 chart. The Tigera operator provides lifecycle management for {{site.prodname}} exposed via the Kubernetes API.
```